### ISSUE TITLE
New library: GSMSim

### ISSUE THREAD

**Issue by erdemarslan** [arduino/Arduino#6699]
Hi,
I wrote a new library for gsm modules. This library tested on SIMCOM modules. Please add this library to the Arduino Library Manager.
https://github.com/erdemarslan/GSMSim
Thanks.

**Comment by per1234** [comment]
Your repository is not compliant with the requirements for inclusion in the Library Manager index:
1. Add a library.properties file to the root of the repository, as described [here](https://github.com/arduino/Arduino/wiki/Arduino-IDE-1.5:-Library-specification#libraryproperties-file-format).
2. Create a release or tag that matches the `version` value in your library.properties file.

After doing so, please comment here to let us know we can proceed with adding your library to the Library Manager index.

For more information see:
https://github.com/arduino/Arduino/wiki/Library-Manager-FAQ

**Comment by erdemarslan** [comment]
Sorry. I checked and solved problems. Can you check again?
Thanks.

**Comment by per1234** [comment]
You still need to create a 1.0.0 release or tag. It looks like you created a branch named 1.0.0 instead.

**Comment by erdemarslan** [comment]
I created release 1.0.19 which is same as in library.properties file. Thanks.

**Comment by erdemarslan** [comment]
Hi my friend.
Are you check it again?
Thanks.

**Comment by per1234** [comment]
Yes, it's fine now. cmaglie will add it to the Library Manager index on the next run through the outstanding requests.
